docs/source/tutorials/upload.rst

```diff
@@ -0,0 +1,104 @@
+
+On SHERLOCK2 you can go to the directory and run::
+
+  module load python/3.6.1
+  . CATKITENV/bin/activate
+
+to use the local installation of CatKit. Alternatively you can install your own version of CatKit - see instructions below.
+
+Installing CatKit
+...........................
+CatKit is set of computational tools for catalysis, that comes with a cli tool :``cathub`` that will be used to arrange your data into folders and submit your data to the server. To install, use pip::
+
+  pip install --upgrade --user git+https://github.com/kirstenwinther/CatKit.git#egg=catkit
+
+which will install CatKit and all the dependencies.
+
+To test that the cathub cli is working, start by typing::
+
+  cathub
+
+And you should see a list of subcommands. If it's not working you probably have to add the installation path to :code:`PATH` in your ~/.bashrc. This would typically be :code:`export PATH=~/.local/bin:${PATH}` for Linux, and :code:`export PATH~/Library/PythonX.Y/bin:${PATH}` for Mac.
+
+You have two options for organizing your data:
+
+* cathub organize: For larger systematic datasets without reaction barriers, this approach will create folders and and arrange your data-files in the right location for you.
+  
+* cathub make_folders: For smaller or more complicated datasets with reaction barriers, this method will only create your folders, and you will have to drop the files in the right location yourself. It's recommended to write a script to transfer files from one folder-structure to another in a systematic way, for example using :code:`shuttils.copyfile('/path/to/initial/file', '/path/to/final/file')`. 
+
+cathub organize
+................
+This tool will take all your structure files from a general folder and organize them in the right folder-structure that can be used for data submission. Note: this approach does not work for transition states / barrier calculations. 
+  
+To learn about the organize command, type::
+  
+  cathub organize --help
+
+To read the data from a general folder, type::
+  
+  cathub organize <FOLDER> -a ADS1,ADS2 -f <FACET>
+
+Use the -a option to specify which adsorbates to look for. Also, please use the -f option to specify the surface facet when applicable. This will generate an organized folder named ``<FOLDER>.organized``. Please open the .txt file ``<FOLDER>.organized/publication.txt``, and update it with info about your publication. It should look something like this::
+  
+  { 
+    "volume": "8", 
+    "publisher": "",
+    "doi": "10.1002/cssc.201500322", 
+    "title": "The Challenge of Electrochemical Ammonia Synthesis: A New Perspective on the Role of Nitrogen Scaling Relations",
+    "journal": "ChemSusChem",
+    "authors": ["Montoya, Joseph H.", "Tsai, Charlie", "Vojvodic, Aleksandra", "Norskov, Jens K."],
+    "year": "2015",
+    "number": "13",
+    "tags": [],
+    "pages": "2140-2267"
+  }
+Note that authors should be a list, with names in the form "lastname, firstname M.".
```

Comment:
Add: "Please go through the created folder and rename folders to make your data easier to localize later. For example a structure folder like Pt16_structure, could be changed to Pt16_fcc or Pt16_bcc respectively. Please do not use spaces in any folders or folders."